### ISSUE TITLE
CRA commit request:  include dotenv-expand

### ISSUE THREAD

**Issue by manuphatak** [CodingZeal/create-react-app#30]
Hello, 

I have a request! :)

In the CRA master branch, they added a package: `dotenv-expand`.  

https://github.com/facebook/create-react-app/blob/836bb39394bef9d83d4dd0cccea08c156dbf298f/packages/react-scripts/config/env.js#L36-L49

This change would be really useful for us if it got merged to this fork

### use case

Recently we started using storybook a bunch.  CRA automatically loads env variables prefixed with `REACT_APP_` and storybook loads variables prefixed with "STORYBOOK_".  Our `.env` file IS checked in to git.  So it'd be really convenient if we could do:

```
# .env
REACT_APP_CUSTOM_ID = $CUSTOM_ID
STORYBOOK_CUSTOM_ID = $CUSTOM_ID
```

This way our devs only have to set ONE variable in `.env.local` or on our server and it would be available across our entire stack.  Right now, I'm setting 3 variables for an external API, which ends up being 9 variables.  And involves some ugly duplication on our heroku env.

**Comment by randycoulman** [comment]
It looks like that change comes with 1.1.0, and we haven't updated our fork to that version yet.  We'll do that as soon as we have time to get to it.

**Comment by frank-west-iii** [comment]
@bionikspoon, we released a new version that bumps us to CRA 1.1.4

**Comment by manuphatak** [comment]
woohoo!  thanks y'all!
